tests/TestCase/Log/Engine/BaseLogTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         t.b.d.
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Log\Engine;
+
+use Cake\Log\Engine\BaseLog;
+use Cake\ORM\Entity;
+use Cake\TestSuite\TestCase;
+use Psr\Log\LogLevel;
+
+/**
+ * Class BaseLogImpl
+ * Implementation of abstract class {@see Cake\Log\Engine\BaseLog},
+ * required by test case {@see Cake\Test\TestCase\Log\Engine\BaseLogTest}.
+ */
+class BaseLogImpl extends BaseLog
```

Comment:
@markstory Fair point. On the other hand the inconsistent handling of unicode is implemented in `BaseLog` and not in the sub classes. Therefore I decided for this pragmatic approach for testing the behaviour.

Perhabs it is a good idea to extract the `_format()` function into some kind of LogFormatter class that could be tested independently.

But for now I hope that this hands-on approach for testing the change is good enough.